openfl-gramine/Dockerfile.gramine

```diff
@@ -5,7 +5,8 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN --mount=type=cache,target=/root/.cache/ \
     pip install --upgrade pip && \
-    pip install openfl
+    git clone --single-branch --branch develop https://github.com/intel/openfl.git && \
+    pip install ./openfl
```

Comment:
Please remove this fix as we will introduce a flexible `development mode` instead